### PR TITLE
WIP: Initial commit

### PR DESCRIPTION
**Pull request by seifmegahed**

### CODE REVIEW

File: bom/README.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+| Part  | Value                                                                                                              | Device                                | Package              | Farnell ID                           | Mouser ID | Man          |
```
Comment (RyanMeulenkamp) [inline bom/README.md]: Why not just call this file `BOM.md`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (romni) [inline README.md:1]: should have a header like:

```
# motherboard
> generic motherboard for robotic applications
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+![](./images/MB.png)
+
+# System Overview.
```
Comment (romni) [inline README.md]: trailing period should be removed

File: README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+The Motherboard has seven identical slots for ![](https://latex.codecogs.com/gif.latex?35mm%20%5Ctimes%2040mm) modules which
+are connected to a single CPU module slot which has up to seventy general purpose
+inputs and outputs (GPIO), as well as ![](https://latex.codecogs.com/gif.latex?SPI), ![](https://latex.codecogs.com/gif.latex?I%5E%7B2%7DC), and ![](https://latex.codecogs.com/gif.latex?USB) communication buses.
```
Comment (RyanMeulenkamp) [inline README.md]: Although, I'm a fan of latex, if you just want to use it to create *italics* or **strong emphasis**, putting it between  asterisks, or underscores (single our double respectively) also works.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+
+![](./images/MB.png)
+
+# System Overview.
+
+The Motherboard has seven identical slots for ![](https://latex.codecogs.com/gif.latex?35mm%20%5Ctimes%2040mm) modules which
```
Comment (romni) [inline README.md]: i think you could put the math inside the description: `35mm \times 40mm`

**Comment by romni** [comment]
i'm thinking the files named `motherboard_g4_V1.3.xxx` should be renamed to `motherboard.xxx`. what do you think?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+**Attribution** — You must give [appropriate credit](/license/credit.md), provide a link to the license, and
+[indicate if changes were made](https://wiki.creativecommons.org/License_Versions#Modifications_and_adaptations_must_be_marked_as_such).
+You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+
+**NonCommercial** — You may not use the material for [commercial purposes]().
+
+**ShareAlike** — If you remix, transform, or build upon the material, you must distribute your contributions under the [same license](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode) as the original.
+
+![](https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png)
```
Comment (RyanMeulenkamp) [inline README.md]: I miss a simple note clarifying that these are `Eagle` designs. `KiCad` uses the same file extensions.

**Review by RyanMeulenkamp — CHANGES_REQUESTED** [review]

**Comment by RyanMeulenkamp** [comment]
I agree with @romnous. Versions should not be in the repo. Git and releases take care of that.

**Review by RyanMeulenkamp — APPROVED** [review]

**Review by romni — DISMISSED** [review]

**Review by romni — APPROVED** [review]

**Pull request merged**